packages/flutter/test/painting/text_painter_rtl_test.dart

```diff
@@ -646,7 +646,7 @@ void main() {
     expect(painter.preferredLineHeight, 100.0);
 
     expect(
-      painter.getBoxesForSelection(const TextSelection(baseOffset: 0, extentOffset: 2)),
+      painter.getBoxesForSelection(const TextSelection(baseOffset: 0, extentOffset: 3)),
       const <TextBox>[
         const TextBox.fromLTRBD(  0.0,  80.0, 100.0, 180.0, TextDirection.ltr),
         const TextBox.fromLTRBD(100.0, 152.0, 110.0, 162.0, TextDirection.ltr),
```

Comment:
the comment on line 656 should also be removed.

Reply:
done